api/locale/ba.yml

```
@@ -0,0 +1,20 @@
+ba:
+  views: "%{number} বার দেখা হয়েছে"
+  seconds-ago:
+    one: "1 সেকেন্ড আগে"
+    many: "%{count} সেকেন্ড আগে"
+  minutes-ago:
+    one: "1 মিনিট আগে"
+    many: "%{count} মিনিট আগে"
+  hours-ago:
+    one: "1 ঘন্টা আগে"
+    many: "%{count} ঘন্টা আগে"
+  days-ago:
+    one: "1 দিন আগে"
+    many: "%{count} দিন আগে"
+  months-ago:
+    one: "1 মাস আগে"
+    many: "%{count} মাস আগে"
+  years-ago:
+    one: "1 বছর আগে"
+    many: "%{count} বছর আগে"
```

It seems like YouTube translates it like this. I am not familiar with the language as much, so if you think this translation is just as good, feel free to commit the suggestion

```suggestion
    one: "1 বছর পূর্বে"
    many: "%{count} বছর পূর্বে"
```